### PR TITLE
feat35/ Add Image support to Amazon Nova

### PR DESCRIPTION
**Pull request by LeonRuggiero**
Adds image support for inputs to Amazona Nova

* https://github.com/becomposable/llumiverse/issues/35

Supported file types are "jpeg" | "png" | "gif" | "webp"

### CODE REVIEW

File: core/src/formatters/nova.ts [hunk 1]
```diff
@@ -56,17 +51,26 @@ export async function formatNovaPrompt(segments: PromptSegment[], schema?: JSONS
     for (const segment of segments) {
 
         const parts: NovaMessagePart[] = [];
-        /*
         if (segment.files) for (const f of segment.files) {
             //TODO type: 'image' -> detect from f.mime_type
```
Comment (mincong-h) [inline core/src/formatters/nova.ts:55]: done?

File: core/src/formatters/nova.ts [hunk 2]
```diff
@@ -13,16 +13,11 @@ export interface NovaSystemMessage{
...
-        type: "base64",
-        media_type: string,
-        data: string,
-    }, // only set for images
     text?: string // only set for text messages
     image?: {
         format: "jpeg" | "png" | "gif" | "webp",
         source: {
-            bytes: "base64",
+            bytes: string //"base64",
```
Comment (mincong-h) [inline core/src/formatters/nova.ts:20]: are these changes intended?

Comment (LeonRuggiero) [reply core/src/formatters/nova.ts:20]: Yes, most of this code is copied from claude.ts where there was a fixed value property with the value "base64". Base64 is not a type, the property which contains the base64 is of type string. In my initial copy paste over from claude.ts I just had it as "base64", string is correct.

File: core/src/formatters/nova.ts (hunk 1 above)
Comment (LeonRuggiero) [reply core/src/formatters/nova.ts]: Thanks for noticing, comments cleaned up.

**Review by mincong-h — APPROVED** [review]
